.github/automation/clang-format.sh

```diff
@@ -1,7 +1,8 @@
-#! /bin/bash
+#! /usr/bin/env bash
```

Comment:
Space after shebang must go. I noticed earlier that we have it in a lot of places.

```suggestion
#!/usr/bin/env bash
```

Reply:
Actually it is allowed...